dependency.lic

```diff
@@ -1220,28 +1242,65 @@ def stop_autostart(script)
   end
 end
 
+def set_dev_mode
+  echo "Now turning dev mode on"
+  echo "Developer mode now set to: #{$manager.developer = true}"
+end
+
+def unset_dev_mode
+  echo "Now turning dev mode off"
+  echo "Developer mode now set to: #{$manager.developer = false}"
```

Comment:
I'm not a fan of doing the assignment in the string interpolation, but it won't hold up merging.

Reply:
I don't mind changing this, as I agree it's awkward. Was really just a result of couple of naive find->replace when changing those from custom methods to accessors.